### PR TITLE
Microsoft .NET SDK Preview 6.1.21.30213

### PR DESCRIPTION
**Pull request by tnc1997**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19872)

### CODE REVIEW

File: manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetPreview
+PackageVersion: 6.0.100-preview.5.21302.13
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.1.21.30213
```

File: manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageLocale: en-US
+Publisher: Microsoft
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.locale.en-US.yaml:7]:
```suggestion
Publisher: Microsoft Corporation
```

File: manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#PublisherUrl: 
+#PublisherSupportUrl: 
+#PrivacyUrl: 
+#Author: 
+PackageName: .NET SDK Preview
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.locale.en-US.yaml:12]:
```suggestion
PackageName: Microsoft .NET SDK Preview
```

File: manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetPreview
+PackageVersion: 6.0.100-preview.5.21302.13
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.yaml:5]:
```suggestion
PackageVersion: 6.1.21.30213
```

File: manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetPreview
+PackageVersion: 6.0.100-preview.5.21302.13
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/6.0.100-preview.5.21302.13/Microsoft.dotnetPreview.installer.yaml:5]:
```suggestion
PackageVersion: 6.1.21.30213
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder too.

**Comment by tnc1997** [comment]
> You'll have to update the version folder too.

@ItzLevvie apologies if I'm missing something but where is that suggested version coming from as the version seems to be listed as 6.0.100-preview.5.21302.13 on the download page and release notes?

https://dotnet.microsoft.com/download/dotnet/6.0
https://github.com/dotnet/core/blob/main/release-notes/6.0/preview/6.0.0-preview.5.md

```console
$ dotnet --version
6.0.100-preview.5.21302.13
```

**Comment by ItzLevvie** [comment]
It's coming from Control Panel; and according to the latest Windows Package Manager documentation the version must match the one shown in Control Panel for the best upgrade user experience.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my latest review.

**Comment by tnc1997** [comment]
> It's coming from Control Panel; and according to the latest Windows Package Manager documentation the version must match the one shown in Control Panel for the best upgrade user experience.

Ah, gotcha, apologies again for the confusion!

**Comment by tnc1997** [comment]
> You'll have to update the version folder too.

https://github.com/microsoft/winget-pkgs/pull/19872/commits/7830d7c6c23879f6762dc785c993524ad9f6fadb

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**